teams/teams-ps/teams/Set-CsTeamsShiftsConnectionInstance.md

```diff
@@ -0,0 +1,349 @@
+```
+```output
+{
+    "id": "WCI-648a8c8f-0ca3-460b-b71c-0d038d6d6e09",
+    "tenantId": "dfd24b34-ccb0-47e1-bdb7-e49db9c7c14a",
+    "name": "MyInstance-Updated",
+    "connector": {
+        "id": "6A51B888-FF44-4FEA-82E1-839401E9CD74",
+        "name": "Blue Yonder V1"
+    },
+    "connectorSpecificSettings": {
+        "adminApiUrl": "https://nehstdevwfm02.replgroup.com/retail/data/wfmadmin/api/v1-beta2",
+        "siteManagerUrl": "https://nehstdevfas01.replgroup.com/retail/data/wfmsm/api/v1-beta2",
+        "essApiUrl": "https://nehstdevfas01.replgroup.com/retail/data/wfmess/api/v1-beta1",
+        "retailWebApiUrl": "https://nehstdevwfm02.replgroup.com/retail/data/retailwebapi/api/v1",
+        "cookieAuthUrl": "https://nehstdevwfm02.replgroup.com/retail/data/login",
+        "federatedAuthUrl": "https://nehstdevfas01.replgroup.com/retail/data/login"
+    },
+    "enabledConnectorScenarios": [
+        "Shift",
+        "SwapRequest",
+        "UserShiftPreferences",
+        "OpenShift",
+        "OpenShiftRequest",
+        "TimeOff",
+        "TimeOffRequest"
+    ],
+    "workforceIntegrationId": "WFI_fe5e773e-784d-4a19-be2b-808dde071d88",
+    "enabledWfiScenarios": [
+        "SwapRequest",
+        "OpenShiftRequest",
+        "TimeOffRequest"
+    ],
+    "syncFrequencyInMin": 10,
+    "designatedActorId": "5d5eaa2d-422c-4d4b-a2db-4cea80f98255",
+    "etag": "\"28007861-0000-0400-0000-60d3579b0000\""
+}
+```
+
+Updates the instance with ID `WCI-648a8c8f-0ca3-460b-b71c-0d038d6d6e09` with name `MyInstance-Updated`. Returns the object of created connector instance.
+
+## PARAMETERS
+
+### -ConnectorInstanceId
+
+The ID of the connection instance.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+Applicable: Microsoft Teams
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DesignatedActorId
+
+The ID of the connection instance creator. You can retrieve user ID from Azure portal or Graph Explorer.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+Applicable: Microsoft Teams
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -IfMatch
+
+The Etag of connection instance. Note: this will be changed after each update of the connection instance. You can get it from instance details using [Get-CsTeamsShiftsConnectionInstance](Get-CsTeamsShiftsConnectionInstance.md)
```

Comment:
```suggestion
The Etag of connection instance. Note: this will be changed after each update of the connection instance. You can get it from instance details using [Get-CsTeamsShiftsConnectionInstance](Get-CsTeamsShiftsConnectionInstance.md).
```